ansible/templates/usb/scripts/prompt-config.sh.j2

```diff
@@ -1347,6 +1362,18 @@ console=$(getanswer "config_console")
 # prompt:
 if [[ -z ${console} || $(tty) != "/dev/console" ]]; then
 	read continue;
+	# automatic installation is always advanced
+	printheader "Advanced installation"
+	printf "
+The advanced installation will allow you to customize additional network
+options, such as VLANs, multiple IP addresses, NTP servers, etc.
+Basic installation is suffcient for most installations\n\n"
+	promptbool "\nEnable advanced installation? (y/n)" "n"
```

Comment:
Why is there `\n` at the begging of the question?

Please add a third parameter (probably with the value `advanced_install`) to support the answer_file feature.

Also please consider changing the selection of the default value to something like this:
```
    [[ "${advanced_install}" -eq 1 ]] && advanced_install_def="y" || advanced_install_def="n"
    promptbool "Enable advanced installation? (y/n)" "${advanced_install_def}" "advanced_install"
```
This way, we can change the global setting when we decide that the advanced installation should be enabled by default.